src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,5 @@
+<header th:fragment="navbar">
+    <a href="#">My Places</a>
+    <a href="#">Search</a>
+    <a href="#">Settings</a>
+</header>
```

Comment:
I chose to extract your navigation bar and put it into a separate HTML file so you can reuse it on multiple pages and can be easily edited in one place.

Source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3